man/ostree.xml

```diff
@@ -290,6 +290,14 @@ Boston, MA 02111-1307, USA.
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><citerefentry><refentrytitle>ostree-find-remotes</refentrytitle><manvolnum>1</manvolnum></citerefentry></term>
```

Comment:
Technically this change should be conditional on `--enable-experimental-api`. Doing that would be a right pain, so I suggest just commenting it out with `<!-- FIXME: Enable this once find-remotes is not experimental` until later.